app/views/placements/providers/_primary_navigation.html.erb

```diff
@@ -1,9 +1,13 @@
 <%# locals: (current_navigation: nil, provider:) -%>
-<% if current_user.organisation_count > 1 %>
+<% if current_user.support_user? || current_user.organisation_count > 1 %>
   <%= content_for(:header_content) do %>
     <%= render(ContentHeaderComponent.new(
       title: provider.name,
-      actions: [govuk_link_to(t(".change_organisation"), placements_organisations_path, no_visited_state: true)],
+      actions: [govuk_link_to(
+        t(".change_organisation"),
+        current_user.support_user? ? placements_support_organisations_path : placements_organisations_path,
+        no_visited_state: true,
+      )],
     )) %>
   <% end %>
 <% end %>
```

Comment:
This could be a good candidate for refactoring as an 'organisation switcher' View Component:

- The [conditional rendering logic](https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/views/placements/providers/_primary_navigation.html.erb#L2) would work well as a `#render?` method
- The [link path logic](https://github.com/DFE-Digital/itt-mentor-services/blob/89874b12bb0a01ed3526cee2da96fc57a7ac71c1/app/views/placements/providers/_primary_navigation.html.erb#L8) would also fit nicely into a component class
- ...which would then be easier to unit test in a component spec

Anyway, probably another thing that's out of scope for this PR 🙂